site/api/prisma/schema.prisma

```diff
@@ -348,19 +385,52 @@ model GroupAlias {
   @@map("group_aliases")
 }
 
+model Company {
+  id           Int       @id @default(autoincrement())
+  name         String
+  creationDate DateTime? @map("creation_date")
+  groups       Group[]
+
+  createdAt DateTime @default(now()) @map("created_at")
+  updatedAt DateTime @default(now()) @updatedAt @map("updated_at")
+  @@map("companies")
+}
+
 model Group {
-  id         Int           @id @default(autoincrement())
-  name       String
-  bannerId   Int?          @unique(map: "groups.banner_id_unique") @map("banner_id")
-  avatarId   Int?          @unique(map: "groups.avatar_id_unique") @map("avatar_id")
-  ireneBotId Int?          @unique(map: "groups.irene_bot_id_unique") @map("irene_bot_id")
-  createdAt  DateTime      @default(now()) @map("created_at")
-  updatedAt  DateTime      @default(now()) @updatedAt @map("updated_at")
-  avatar     Image?        @relation("group_avatar", fields: [avatarId], references: [id])
-  banner     Image?        @relation("group_banner", fields: [bannerId], references: [id])
-  aliases    GroupAlias[]
-  members    GroupMember[]
+  id           Int           @id @default(autoincrement())
+  name         String
+  bannerId     Int?          @unique(map: "groups.banner_id_unique") @map("banner_id")
+  avatarId     Int?          @unique(map: "groups.avatar_id_unique") @map("avatar_id")
+  ireneBotId   Int?          @unique(map: "groups.irene_bot_id_unique") @map("irene_bot_id")
```

Comment:
do we really want it to be reliant on irene? what if that bot goes offline? 

What I mean is similiar to a image provider:

Do we want to make this reusable, so we are agnostic to the provider we use?

It just has a concious decision we make

Reply:
this doesn't rely on irene. It just helps keep track of which data is already added from irene bot. We can just get rid of the column once all the data is moved over and we have people editing data directly on kiyomi

Reply:
Okidoki.. because this wouldnt fit the agnostic approach you are going for.. but if its just for migration purposes..